common/scripts/base.sh

```diff
@@ -139,3 +138,23 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
+{
+  if ! bash ${mesos_artifacts_home}/common/${1}/deploy.sh; then
+    echoError "Non-zero exit code returned when deploying ${1}"
+  fi
+}
+
+function deploy_wso2_service()
```

Comment:
@Thanu I don't think we need "wso2" part here because we are using this function to deploy both wso2 products and dbs. Any service deployed is going to be "wso2" specific service anyway. I think it is redundant.